docs/src/languages/php/redis.md

```diff
@@ -6,103 +6,49 @@ weight: 7
 
 If the version of the PhpRedis extension available for your PHP version is not compatible with your application and upgrading your application is not feasible, you can use the script below as an alternative to download and compile a precise version of the extension on the fly.
 
 Do *not* use this approach unless you really need to.  Using the provided PhpRedis extension is preferred in the vast majority of cases.
 
-## Using the Redis builder script
-
-1. Copy the following script into a file named `install-redis.sh` in [your app root](../../create-apps/app-reference.md#root-directory)
-
-```bash
-run() {
-    # Run the compilation process.
-    cd $PLATFORM_CACHE_DIR || exit 1;
-
-    if [ ! -f "${PLATFORM_CACHE_DIR}/phpredis/modules/redis.so" ]; then
-        ensure_source
-        checkout_version "$1"
-        compile_source
-    fi
-
-    copy_lib
-    enable_lib
-}
-
-enable_lib() {
-    # Tell PHP to enable the extension.
-    echo "Enabling PhpRedis extension."
-    echo -e "\nextension=${PLATFORM_APP_DIR}/redis.so" >> $PLATFORM_APP_DIR/php.ini
-}
-
-copy_lib() {
-    # Copy the compiled library to the application directory.
-    echo "Installing PhpRedis extension."
-    cp $PLATFORM_CACHE_DIR/phpredis/modules/redis.so $PLATFORM_APP_DIR
-}
-
-checkout_version () {
-    # Check out the specific Git tag that we want to build.
-    git checkout "$1"
-}
-
-ensure_source() {
-    # Ensure that the extension source code is available and up to date.
-    if [ -d "phpredis" ]; then
-        cd phpredis || exit 1;
-        git fetch --all --prune
-    else
-        git clone https://github.com/phpredis/phpredis.git
-        cd phpredis || exit 1;
-    fi
-}
-
-compile_source() {
-    # Compile the extension.
-    phpize
-    ./configure
-    make
-}
-
-ensure_environment() {
-    # If not running in a Platform.sh build environment, do nothing.
-    if [ -z "${PLATFORM_CACHE_DIR}" ]; then
-        echo "Not running in a Platform.sh build environment.  Aborting Redis installation."
-        exit 0;
-    fi
-}
-
-ensure_arguments() {
-    # If no version was specified, don't try to guess.
-    if [ -z $1 ]; then
-        echo "No version of the PhpRedis extension specified.  You must specify a tagged version on the command line."
-        exit 1;
-    fi
-}
-
-ensure_environment
-ensure_arguments "$1"
-run "$1"
-```
-
-2. Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
+A [script](https://github.com/platformsh/snippets/blob/main/src/install-phpredis.sh) is provided to ease the installation of a customer version of PhpRedis.  Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
```

Comment:
:hammer_and_wrench: We prefer links at the ends of sentences: https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#link-at-the-end-of-sentences-in-sentence-case

```suggestion
To ease the installation of a customer version of PhpRedis, use a [PhpRedis install script](https://github.com/platformsh/snippets/blob/main/src/install-phpredis.sh).
Invoke this script from your build hook, specifying a version.
Any tagged version of the library is acceptable:
```